templates/cloudrun.yml

```diff
@@ -104,6 +95,29 @@
         COMMAND_TO_RUN+=( --set-env-vars=${ENV} )
       fi
 
+      if [ -n "${SECRET_YAML}" ]; then
+
+        echo
+        echo "-> ... also deploying kubernetes secrets"
+        echo
+
+        echo "${SECRET_YAML}" > kube/secret.yaml
+        kubectl apply -f kube/secret.yaml --namespace=${NAMESPACE}
+
+        COMMAND_TO_RUN+=( --set-secrets=[${SECRET_MOUNTS}] )
+      fi
+
+      if [ -f "kube/configmap.yaml" ]; then
+
+        echo
+        echo "-> ... also deploying kubernetes configmaps"
+        echo
+
+        kubectl apply -f kube/configmap.yaml --namespace=${NAMESPACE}
```

Comment:
I think that we must add a variable for the configmap path, kube/configmap.yaml could be the default value

Reply:
do you think this is solved this way?